python/usage_example.py

```diff
@@ -0,0 +1,49 @@
+import mHT 
+from mHT.CPF import cpf_fast, cpf_accurate
+x=1; #dimensionless
+y=1; #dimensionless
+print("the output of the cpf_accurate function")
+print cpf_accurate(x,y)

```

Comment:
Python 2 syntax, causes an error in Python 3.

 Change to 
`
print(cpf_accurate(x,y))
`